src/esp/gfx/PbrShader.h

```diff
@@ -14,8 +14,13 @@
 
 #include "esp/core/esp.h"
 
-namespace esp {
+namespace Magnum {
+namespace GL {
+class CubeMapTexture;
+};
+};  // namespace Magnum
```

Comment:
`#include <Magnum/GL/GL.h>` instead, it's a header with all forward declarations for the GL namespace. [Docs.](https://doc.magnum.graphics/magnum/compilation-speedup.html#compilation-forward-declarations)

Doing forward declarations manually is *not recommended* because it might cause build breakages in the future (the class might become a typedef and vice versa).

Reply:
>it's a header with all forward declarations
Good. Hope next time you can tell me in your first comment so I can avoid this iteration. :)